radix-engine/src/kernel/kernel.rs

```diff
@@ -319,18 +322,16 @@ where
     M: KernelCallbackObject,
     S: SubstateStore,
 {
```

Comment:
Can we remove tracing for `KernelInternalApi`? They are unstable APIs for special needs of some kernel modules.